### PR TITLE
[inference] Add FusedBiasActKernel 

### PR DESCRIPTION
**Pull request by freeliuzc**
### PR types
New features

### PR changes

OPs
### Description

Pcard-71502  
Add new kernel FusedBiasAct. The kernel includes three optional processes: bias-add, (de)quantization, and activation.

### CODE REVIEW

**Review by tianshuo78520a — APPROVED** [review]

**Review by heavengate — APPROVED** [review]

**Review by kolinwei — APPROVED** [review]

**Review by zhhsplendid — APPROVED** [review]
Approve for data types.

**Review by ZzSean — APPROVED** [review]
LGTM for CI-OP-Benchmark

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

File: test/legacy_test/test_fused_bias_act_op.py [hunk 1]
```diff
@@ -0,0 +1,782 @@
...
+
+import unittest
+
+import numpy as np
+from eager_op_test import convert_float_to_uint16
+from scipy.special import erf, expit
+
+import paddle
+import paddle.nn.functional as F
+from paddle.fluid import core
```
Comment (qingqing01) [inline test/legacy_test/test_fused_bias_act_op.py:23]: do not use fluid API

**Pull request merged**